bifrost/renderer/+config.ts

```diff
@@ -1,28 +1,46 @@
 import { type Config } from "vike/types";
+import { bifrostConfig } from "./configs/bifrost";
+import { wrappedConfig } from "./configs/wrapped";
 
-const passToClient = [
-  "layoutProps",
-  "pageProps",
-  "redirectTo",
-  "bodyAttrs",
-  "documentProps",
-  "scripts",
-  "is404",
-];
 export default {
-  passToClient,
-  onRenderClient: "import:@alignable/bifrost/renderer/onRenderClient",
-  onRenderHtml: "import:@alignable/bifrost/renderer/onRenderHtml",
-  onBeforeRoute: "import:@alignable/bifrost/renderer/onBeforeRoute",
+  name: "@alignable/bifrost",
+  require: {
+    vike: ">=0.4.193",
+  },
+  clientHooks: true,
+
+  onBeforeRoute: "import:@alignable/bifrost/renderer/onBeforeRoute:default",
+  onRenderClient: "import:@alignable/bifrost/renderer/onRenderClient:default",
```

Comment:
because we can only export 1 config, we had to consolidate all logic into a single onRenderClient/onRenderHtml